<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="login">
          <property name="text">
           <string>Admin Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="UsersTabs">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="tabBarAutoHide">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="MainPage">
         <attribute name="title">
          <string>Main Page</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTabWidget" name="MainPageTabs">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="WelcomeTab">
             <attribute name="title">
              <string>Welcome</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="DisplayListsTab">
             <attribute name="title">
              <string>Display Lists</string>
             </attribute>
             <widget class="QPushButton" name="pushButton_2">
              <property name="geometry">
               <rect>
                <x>224</x>
                <y>122</y>
                <width>131</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
             <widget class="QComboBox" name="comboBox">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>70</y>
                <width>331</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
             <widget class="QTableView" name="displayInfo">
              <property name="geometry">
               <rect>
                <x>370</x>
                <y>50</y>
                <width>801</width>
                <height>591</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="calc_label">
              <property name="geometry">
               <rect>
                <x>410</x>
                <y>649</y>
                <width>411</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
             <widget class="QLabel" name="calc_label_2">
              <property name="geometry">
               <rect>
                <x>860</x>
                <y>650</y>
                <width>101</width>
                <height>21</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="calc_label_3">
              <property name="geometry">
               <rect>
                <x>970</x>
                <y>650</y>
                <width>161</width>
                <height>20</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>30</y>
                <width>91</width>
                <height>31</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Sort by:</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="TeamInfoTab">
             <attribute name="title">
              <string>Team Information</string>
             </attribute>
             <widget class="QListWidget" name="listWidget">
              <property name="geometry">
               <rect>
                <x>50</x>
                <y>40</y>
                <width>211</width>
                <height>481</height>
               </rect>
              </property>
             </widget>
             <widget class="QTextEdit" name="textEdit">
              <property name="geometry">
               <rect>
                <x>440</x>
                <y>40</y>
                <width>621</width>
                <height>481</height>
               </rect>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_TeamInfo">
              <property name="geometry">
               <rect>
                <x>290</x>
                <y>230</y>
                <width>121</width>
                <height>71</height>
               </rect>
              </property>
              <property name="toolTip">
               <string>&lt;html class = &quot;solid&quot;&gt;
&lt;head&gt;
&lt;style&gt;
html.solid {border-style: solid;}
&lt;/style&gt;
&lt;/head&gt;
&lt;/html&gt;</string>
              </property>
              <property name="styleSheet">
               <string notr="true">html.solid {border-style: solid;}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="PlanTripTab">
             <attribute name="title">
              <string>Plan Trip</string>
             </attribute>
             <widget class="QPushButton" name="DFS">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>270</y>
                <width>80</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>DFS</string>
              </property>
             </widget>
             <widget class="QPushButton" name="BFS">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>340</y>
                <width>80</width>
                <height>21</height>
               </rect>
              </property>
              <property name="text">
               <string>BFS</string>
              </property>
             </widget>
             <widget class="QTextBrowser" name="TraversalDisplay">
              <property name="geometry">
               <rect>
                <x>240</x>
                <y>10</y>
                <width>951</width>
                <height>621</height>
               </rect>
              </property>
              <property name="font">
               <font>
                <family>Trebuchet MS</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Trebuchet MS'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
             <widget class="QComboBox" name="comboBox_3">
              <property name="geometry">
               <rect>
                <x>30</x>
                <y>80</y>
                <width>72</width>
                <height>22</height>
               </rect>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="AlgorithmTab">
             <attribute name="title">
              <string>Algorithm Test</string>
             </attribute>
             <widget class="QPushButton" name="pushButton_7">
              <property name="geometry">
               <rect>
                <x>200</x>
                <y>190</y>
                <width>181</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Dijkstra’s or the A* algorithm</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_8">
              <property name="geometry">
               <rect>
                <x>200</x>
                <y>220</y>
                <width>181</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Prim’s or Kruskal’s algorithm</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_9">
              <property name="geometry">
               <rect>
                <x>200</x>
                <y>250</y>
                <width>181</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>DFS</string>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButton_10">
              <property name="geometry">
               <rect>
                <x>200</x>
                <y>280</y>
                <width>181</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>BFS</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Admin">
         <attribute name="title">
          <string>Admin</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>14</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Select a Team to modify</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="SelectTeam">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Calibri</family>
                  <pointsize>12</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>300</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableWidget" name="ModifyTable">
              <property name="font">
               <font>
                <family>Calibri</family>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1166</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
